runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -183,7 +183,7 @@ ifeq ($(HOST_ARCH),z)
 endif
 
 ifeq ($(HOST_ARCH),arm)
-    CX_DEFINES+=ARMGNU ARMGNUEABI FIXUP_UNALIGNED HARDHAT
+    CX_DEFINES+=ARMGNU ARMGNUEABI FIXUP_UNALIGNED HARDHAT __STDC_LIMIT_MACROS
```

Comment:
Is this really required for arm?

Reply:
not arm specifically, but whatever combo of gcc/glibc versions we are using on there